pkgs/package-overrides.nix

```diff
@@ -120,6 +120,15 @@ in
                 "ext/dom/tests/bug80268.phpt"
               ];
             })
+          ] ++ lib.optionals (lib.versionOlder prev.php.version "7.3" && lib.versionAtLeast prev.php.version "7.1") [
+            ./patches/php71-libxml212-tests.patch
+          ] ++ lib.optionals (lib.versionOlder prev.php.version "7.4" && lib.versionAtLeast prev.php.version "7.3") [
+            ./patches/php73-libxml212-tests.patch
+          ] ++ lib.optionals (lib.versionOlder prev.php.version "8.1" && lib.versionAtLeast prev.php.version "7.4") [
+            ./patches/php74-libxml212-tests.patch
+          ] ++ lib.optionals (lib.versionOlder prev.php.version "8.2.14" && lib.versionAtLeast prev.php.version "8.1") [
+            # Fix Libxml extension C files
```

Comment:
The patch description should also say what it does, e.g. something like “ Fix compilation errors with libxml2 2.12”

Reply:
Fixed.